test/e2e/lib/jest/globalSetup.ts

```diff
@@ -0,0 +1,47 @@
+import { mkdir } from 'fs/promises';
+import path from 'path';
+import config from 'config';
+import { chromium } from 'playwright';
+
+// Save cookies for defaultUser.
+export default async (): Promise< void > => {
+	if ( process.env.SAVE_AUTH_COOKIES !== 'true' ) {
+		return;
+	}
+
+	const browser = await chromium.launch();
+	const calypsoBaseURL = config.get( 'calypsoBaseURL' );
+	const cookieBasePath = path.join( process.cwd(), 'cookies' );
+	process.env.COOKIES_PATH = cookieBasePath;
```

Comment:
This is ugly, but I don't know a better way to pass info to each test process. Maybe add a comment explaining why we need this here?

Reply:
Comments coming up. I agree that this is not the preferred approach - the separation of `globalXYZ` from rest of the test runner process is an understandable decision by the Jest team, but not something I agree with.